src/App.jsx

```diff
@@ -14,33 +15,79 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+export const App = () => {
+  const copyOfGoods = [...goodsFromServer];
+  const [value, setState] = useState('');
+  const [reserve, setReserve] = useState(false);
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+  copyOfGoods.sort((good1, good2) => {
+    switch (value) {
+      case 'Sort by length':
+        return good1.length - good2.length;
+      case 'Sort alphabetically':
+        return good1.localeCompare(good2);
+      default:
+        return 0;
+    }
+  });
+  if (reserve) {
+    copyOfGoods.reverse();
+  }
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+  const isLight = item => {
+    return value === item ? '' : 'is-light';
+  };
```

Comment:
It is better to rename the function `isLight` to `getButtonClass` and change its implementation.

This improves the readability of the code and reduces its repetition.

Example:

```suggestion
const getButtonClass = buttonType => {
  return sortType === buttonType ? '' : 'is-light';
};
```